zp-relayer/endpoints.ts

```diff
@@ -13,12 +13,22 @@ import {
   checkTraceId,
   validateBatch,
 } from './validation/api/validation'
-import { sentTxQueue, SentTxState } from './queue/sentTxQueue'
 import { HEADER_TRACE_ID } from './utils/constants'
-import { getFileHash } from './utils/helpers'
 import type { FeeManager } from './services/fee'
 
-async function sendTransactions(req: Request, res: Response) {
```

Comment:
why don't just make this method and similar ones that use pool instance a part of Pool class?